lib/rollout/ui/web.rb

```diff
@@ -58,6 +61,7 @@ class Web < Sinatra::Base
             feature.users = params[:users].split(',').map(&:strip).uniq.sort
           end
           feature.data.update(description: params[:description])
+          feature.data.update(updated_at: Time.now)
```

Comment:
will it work as expected without converting to an integer?

Reply:
It works as expected, just because we compare it with ==. But it is good that you have found it, will convert it to_i to have the better format in redis.